run_HamPi_playbook_ubuntu_local

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 - 2021, Dave Slotter (W3DJS). All rights reserved.
```

Comment:
@wsciaroni Please change 2021 to 2022. And you can add your own credits to mine if you'd like for the enhancement.